project-service/tests/helpers.js

```diff
@@ -0,0 +1,118 @@
+import env from 'node-env-file';
```

Comment:
I would say that the convention is to call such files with mock data `fixtures`. Not that it matters...